src/routes/help/components.svelte

```diff
@@ -0,0 +1,242 @@
+	import { copyToClipboard } from '$lib/utils/clipboard';
+	import { extractUnique } from '$lib/utils/extractUnique';
+
+	const tagItems = extractUnique(components, 'tags');
+	const categoryItems = [
+		...extractUnique(components, 'category').filter((cat) => cat.label !== '')
+	];
+
+	let clipboardCopy = false;
+	const copy = () => {
+		copyToClipboard(JSON.stringify(jsonSnippet, null, 4)).then(() => (clipboardCopy = false));
+		clipboardCopy = true;
+	};
+
+	let title = 'svelte-calender';
+	let url = 'https://github.com/6eDesign/svelte-calendar';
+	let description = 'A lightweight date picker with neat animations and a unique UX';
+	let npm = 'svelte-calender';
+	let image = '';
+	let category = 'Forms & User Input';
+	let tags = ['components and libraries', 'time and date'];
+	let addedOn = todaysDate();
+
+	$: jsonSnippet = {
+		title,
+		url,
+		description,
+		npm,
+		tags,
+		image,
+		addedOn,
+		category
+	};
+
+	function padWithZero(date) {
+		return date.toString().padStart(2, '0');
+	}
+
+	function todaysDate() {
+		const date = new Date();
+		const day = padWithZero(date.getDate());
+		const month = padWithZero(date.getMonth() + 1);
+		const year = date.getFullYear();
+		const sep = '-';
+		return [year, month, day].join(sep);
+	}
+</script>
+
+<h1>Submitting a new component</h1>
+<p>
+	To add a new component on the website, the process is rather simple. You have to add your
+	component in <a
+		href="https://github.com/svelte-society/sveltesociety.dev/blob/master/src/routes/components/components.json"
+	>
+		components.json
+	</a>
+</p>
+
+<h2>Editing the file</h2>
+<p>
+	You can edit and propose your changes <a
+		href="https://github.com/svelte-society/sveltesociety.dev/edit/master/src/pages/components/components.json"
+	>
+		directly in GitHub
+	</a>
+</p>
+
+<h2>Generating file contents snippet</h2>
+<p>
+	Each component is represented by a JSON Object. Use the generator below to generate the Object.
+</p>
+
+<p><code>*</code> marked fields are required</p>
+<div class="json-generator">
+	<div class="input-wrapper">
+		<label for="title" class="required">Title:</label>
+		<div>
+			<input id="title" type="text" required bind:value={title} />
+			<span class="input-helper">Name of the component</span>
+		</div>
+	</div>
+	<div class="input-wrapper">
+		<label for="url">URL:</label>
+		<div>
+			<input id="url" type="url" bind:value={url} />
+			<span class="input-helper">The URL where to find it</span>
+		</div>
+	</div>
+	<div class="input-wrapper">
+		<label for="desc">Description:</label>
+		<div>
+			<input id="desc" type="text" bind:value={description} />
+			<span class="input-helper">A short description of the component</span>
+		</div>
+	</div>
+	<div class="input-wrapper">
+		<label for="npm">NPM:</label>
+		<div>
+			<input id="npm" type="text" bind:value={npm} />
+			<span class="input-helper">The npm name of the component</span>
+		</div>
+	</div>
+	<div class="input-wrapper">
+		<label for="img">Image:</label>
```

Comment:
We should remove this. There aren't any. The only instances of it were empty. I just removed them: https://github.com/svelte-society/sveltesociety.dev/commit/91bf13080ddec8031ae400f6bb71ef504dac306c